lib/elixir/lib/application.ex

```diff
@@ -523,10 +523,32 @@ defmodule Application do
     :application.set_env(app, key, value, opts)
   end
 
+  @doc """
+  Puts the environment for multiple apps at the same time.
+
+  It will fail if any of the applications appear more than once
+  or if any of the keys in the application are repeated.
+
+  It receives the same options as `put_env/4`. Returns `:ok`.
+  """
+  @spec put_all_env([{app, [{key, value}]}], timeout: timeout, persistent: boolean) :: :ok
+  def put_all_env(config, opts \\ []) when is_list(config) and is_list(opts) do
+    # TODO: Remove function exported? check when we require Erlang/OTP 22+
+    if function_exported?(:application, :set_env, 2) do
+      :application.set_env(config, opts)
+    else
+      for {app, kw} <- config, {k, v} <- kw do
```

Comment:
kw ?

Reply:
It means "keyword" as in "keyword list". @josevalim if we wanted to be nitpicky I'd probably suggest to call `kw` `keyword` and `k`/`v` `key`/`value` instead.